pkg/services/annotations/annotationsimpl/annotations.go

```diff
@@ -72,12 +72,45 @@ func (r *RepositoryImpl) Update(ctx context.Context, item *annotations.Item) err
 }
 
 func (r *RepositoryImpl) Find(ctx context.Context, query *annotations.ItemQuery) ([]*annotations.ItemDTO, error) {
-	resources, err := r.authZ.Authorize(ctx, query.OrgID, query)
-	if err != nil {
-		return make([]*annotations.ItemDTO, 0), err
+	if query.Limit == 0 {
+		query.Limit = 1000
```

Comment:
I think this is good but should be set and enforced in the API rather than the repo.

Reply:
Good point, makes more sense.